vaadin-testbench-shared/src/main/java/com/vaadin/testbench/TestBench.java

```diff
@@ -86,7 +87,7 @@ public Object invoke(Object self, Method thisMethod, Method proceed,
 
     static {
         LicenseChecker.checkLicenseFromStaticBlock("vaadin-testbench",
-                TestBenchVersion.testbenchVersion, null);
+                TestBenchVersion.testbenchVersion, BuildType.DEVELOPMENT);
```

Comment:
It should be `null` for TestBench to work properly

Reply:
Reverted, wrong license-checker version (1.7.2) was used in tests